src/browser/provider/plugin-host.js

```diff
@@ -0,0 +1,79 @@
+import { assignIn } from 'lodash';
+import Promise from 'pinkie';
+import promisifyEvent from 'promisify-event';
+import BrowserConnection from '../connection';
+
+
+export default class BrowserProviderPluginHost {
+    constructor (providerObject) {
+        assignIn(this, providerObject);
+    }
+
+    // Helpers
+
+    runInitScript (id, code) {
+        var connection = BrowserConnection.getById(id);
+
+        return connection.runInitScript(`(${code})()`);
+    }
+
+    waitForConnectionReady (id) {
+        var connection = BrowserConnection.getById(id);
+
+        if (connection.ready)
+            return Promise.resolve();
+
+        return promisifyEvent(connection, 'ready');
+    }
+    // API
```

Comment:
Here and in many other places - weird blank lines before/after section comments. We add 2 blank lines before section comment, and we don't have blank lines after section comment